common/src/main/spark-3.3/org/apache/comet/shims/ShimBatchReader.scala

```diff
@@ -33,4 +37,24 @@ object ShimBatchReader {
       Array.empty[String],
       0,
       0)
+
+  def getTaskAccumulator(taskMetrics: TaskMetrics): Option[AccumulatorV2[_, _]] = {
```

Comment:
This looks good, but it seems to be the same implementation in all of the shims? Deos it even need to be in the shim layer?

Reply:
I could move this into the `BatchReader` itself. Perhaps that is cleaner.